src/custom/components/AccountDetails/Transaction.tsx

```diff
@@ -120,10 +121,9 @@ function RequestCancellationModal(props: RequestCancellationModalProps): JSX.Ele
           </p>
           {showMore && (
             <>
-              <p>It will be taken into account in a best effort basis.</p>
```

Comment:
Can't add a comment on L115, but is there value in including a link from the short ID to the batch explorer?
```html
 <a href="{batchExplorerLink}"><strong>{shortId}</strong></a>
```

Not sure if this has too much value though - just a random thought.

Reply:
Not a big deal to add. I'm just not sure, since it's available in the previous modal:

![screenshot_2021-06-08_15-03-11](https://user-images.githubusercontent.com/43217/121540176-1dbe2500-c9bb-11eb-88b5-fdd5dd05f56c.png)
Every entry here links to the respective explorer page.
`1` is a pending order that you can cancel, which will lead you to the modal in question.

I'm not strong against it though.

@biocom what are your thoughts on this?

Reply:
> since it's available in the previous modal:

Ah, in that case it shouldn't be needed then.